src/server/services/dialog.js

```diff
@@ -16,18 +16,113 @@ const Client = new Messenger.Client(CONFIG);
 
 /**
  * @arwa Let's put all the messenge sending logic to FB messenger here
+ * @team, this is the basics of the message sending logic:
+ * 
+ * 1. Chatbot initiates intro dialog. Then Offers options for user choice : 1. place order 2. check order
+ * 2.1 place order:
+ *   2.1.1 chatbot asks for location
+ *   2.1.2 user returns location
+ *   2.1.3 chatbot returns url links to F&B places nearby. these will link to the merchant webviews.
+ *   2.1.4 user selects url to preferred F&B place, and does ordering in merchant webview.
+ * 
+ * 2.2 check order
+ *   2.2.2 chatbot pulls out order and returns to user
+ *   2.2.3 The user will be returned with a receipt
+ * 
  * 
  */
 
-function introduction(psid, customer) {
+
+
+function introduction(psid, customer, text) {
   // @todo Sends the introduction message when Customers first initiate chat
   const recipient = {'id': psid};
   Client.sendText(recipient, ResponseTemplates.Introduction(customer.name));
+
+  let messageData = {
+    "message":{
+      "text": "Hello, thanks for getting in touch."
+    },
+    "message":{
+      "text":"How can we help you today?"
+    }
+  }
+
+  sendRequest(psid, customer, messageData)
```

Comment:
Am I missing something but I can't find a "sendRequest" method anywhere

Reply:
I thought sendRequest was an in-built class, but i may be wrong:
https://developers.facebook.com/docs/php/Facebook/5.0.0

Reply:
as discussed in the general discussion thread, please follow that repo